### ISSUE TITLE
filmstoon.xyz

### ISSUE THREAD

**Issue by smed79** [abp-filters/abp-filters-anti-cv#399]
**Website**

* `https://www.filmstoon.pro/`

**Problem**

* popup

**List all of your filter lists**

- ABP filters
- EasyList + FR
- EasyPrivacy

**Fix**

```
filmstoon.pro,filmstoon.xyz,filmstreaming1.pro,filmstreaming1.co,filmstreaming1.com,filmstreaming1.plus,filmstreaming1.rip#$#abort-current-inline-script Math zfgloaded
```

**Comment by wizmak** [comment]
> filmstoon.pro,filmstoon.xyz,filmstreaming1.pro,filmstreaming1.co,filmstreaming1.com,filmstreaming1.plus,filmstreaming1.rip#$#abort-current-inline-script Math zfgloaded

Doesn't work for me. Should be closed in 1a6b0f3.

**Comment by wizmak** [comment]
> > filmstoon.pro,filmstoon.xyz,filmstreaming1.pro,filmstreaming1.co,filmstreaming1.com,filmstreaming1.plus,filmstreaming1.rip#$#abort-current-inline-script Math zfgloaded
> 
> Doesn't work for me. Should be closed in [1a6b0f3](https://github.com/abp-filters/abp-filters-anti-cv/commit/1a6b0f325114df58c363c84a4d2b7fc768e4e4e0).

@smed79 let me know if your suggested fix still works for you, I will then add it anyways.

**Comment by smed79** [comment]
Fixed in Liste FR.
